src/components/networks/createNetworkDialog.tsx

```diff
@@ -218,39 +281,67 @@ const IpRow = ({ onValueChanged, dialogValues, validationFailed }) => {
     );
 };
 
-const DhcpRow = ({ ipVersion, rangeStart, rangeEnd, expanded, onValueChanged, validationFailed }) => {
-    const validationStart = validationFailed['ipv' + ipVersion + 'DhcpRangeStart'] ? 'error' : 'default';
-    const validationEnd = validationFailed['ipv' + ipVersion + 'DhcpRangeEnd'] ? 'error' : 'default';
+const DhcpRow = ({
+    ipVersion,
+    rangeStart,
+    rangeEnd,
+    expanded,
+    onRangeStartChanged,
+    onRangeEndChanged,
+    onExpandedChanged,
+    validationFailedRangeStart,
+    validationFailedRangeEnd,
+} : {
+    ipVersion: string,
+    rangeStart: string,
+    rangeEnd: string,
+    expanded: boolean,
+    onRangeStartChanged: (val: string) => void,
+    onExpandedChanged: (val: boolean) => void,
+    onRangeEndChanged: (val: string) => void,
+    validationFailedRangeStart: string | undefined,
+    validationFailedRangeEnd: string | undefined,
+}) => {
+    const validationStart = validationFailedRangeStart ? 'error' : 'default';
+    const validationEnd = validationFailedRangeEnd ? 'error' : 'default';
 
     return (
         <>
             <FormGroup>
                 <Checkbox id={'create-network-ipv' + ipVersion + '-dhcp'}
                           isChecked={expanded}
                           label={_("Set DHCP range")}
-                          onChange={() => onValueChanged('ipv' + ipVersion + 'DhcpEnabled', !expanded)} />
+                    onChange={() => onExpandedChanged(!expanded)} />
```

Comment:
Indendation verschlimmbessert -- desert is cancelled for our linters! :cupcake: :stop_sign:

Reply:
Ouch!